frontend/src/views/Home.vue

```diff
@@ -64,6 +64,46 @@ export default {
       additionalInfo: '',
     }
   },
+
+  created() {
+    const hasOrgDetails = this.$keycloak.tokenParsed.hasOwnProperty('org_details')
+    const hasOldLdapId = this.$keycloak.tokenParsed.hasOwnProperty('old_ldap_id')
+    let roleExist = false
+    let audience = ''
+
+    // Check if LDAP account has already been transferred and old uid exists in keycloak
+    if (hasOldLdapId) {
+      this.$router.push({
+        name: 'AlreadyTransferred',
+      })
+      return
+    }
+    // Check if audience exists
+    if (this.$keycloak.tokenParsed.hasOwnProperty('aud')) {
+      const aud = this.$keycloak.tokenParsed.aud
+      if (typeof aud === 'string') {
+        audience = aud.startsWith('MSPDIRECT-SERVICE') ? aud : ''
+      } else {
+        audience = aud.find((element) => element.startsWith('MSPDIRECT-SERVICE'))
+      }
+    }
+    // Check if role has been transferred
+    if (this.$keycloak.tokenParsed.hasOwnProperty('resource_access')) {
+      if (this.$keycloak.tokenParsed.resource_access.hasOwnProperty(audience)) {
+        const mspDirect = this.$keycloak.tokenParsed.resource_access[audience]
+        if (mspDirect.roles && mspDirect.roles.length > 0) {
+          roleExist = true
+        }
+      }
+    }
+    // Check if LDAP Account has partially been transferred
+    if ((roleExist && !hasOrgDetails) || (!roleExist && hasOrgDetails)) {
+      const warningMessage = 'Your MSP Direct Account has been partially transferred. Please complete the transfer below. If this error persists, please contact <a href="https://HLTH.HelpDesk.hlth.gov.bc.ca" target="_blank">HLTH.HelpDesk.gov.bc.ca</a>'
```

Comment:
I believe this should be an email contact and not a url? That url isn't valid
hlth.helpdesk@gov.bc.ca

e.g.)
please contact the helpdesk at <b>(250) 952-1234</b> or <b>hlth.helpdesk@gov.bc.ca</b>